lib/audited/auditor.rb

```diff
@@ -232,7 +243,23 @@ def require_comment
       end
 
       def auditing_enabled
-        self.class.auditing_enabled
+        if_condition = audited_options[:if]
+        unless_condition = audited_options[:unless]
+        return self.class.auditing_enabled unless if_condition.present? || unless_condition.present?
```

Comment:
This check is redundant, since the first check in `run_conditional_check` returns true if the condition is not present.